lottie/CMakeLists.txt

```diff
@@ -14,9 +14,6 @@ set(CMAKE_AUTOUIC ON)
 if(NOT CMAKE_PROJECT_NAME STREQUAL PROJECT_NAME)
     set(CMAKE_FOLDER "Lottie")
 endif()
-if(${CMAKE_VERSION} VERSION_GREATER_EQUAL 3.17)
-    cmake_policy(SET CMP0099 OLD)
-endif()
```

Comment:
Why remove this?

Reply:
Gela just asked me the same question, see the answer [here](https://github.com/mozilla-mobile/mozilla-vpn-client/pull/8324#discussion_r1364067283)